astacus/coordinator/plugins/base.py

```diff
@@ -17,6 +17,7 @@
 from astacus.coordinator.manifest import download_backup_manifest, download_backup_min_manifest
 from collections import Counter
 from collections.abc import Sequence, Set
+from starlette.concurrency import run_in_threadpool
```

Comment:
I see no starlette dependency neither in astacus nor in aiven-core. I can imagine there's a transitive dependency, but if we start using it directly we should declare a direct dependency.

Reply:
You're right it was already there, it wasn't so urgent.